bazel/ray_deps_setup.bzl

```diff
@@ -250,11 +250,11 @@ def ray_deps_setup():
         # https://github.com/grpc/grpc/blob/1ff1feaa83e071d87c07827b0a317ffac673794f/bazel/grpc_deps.bzl#L189
         # Ensure this rule matches the rule used by grpc's bazel/grpc_deps.bzl
         name = "boringssl",
-        sha256 = "e168777eb0fc14ea5a65749a2f53c095935a6ea65f38899a289808fb0c221dc4",
-        strip_prefix = "boringssl-4fb158925f7753d80fb858cb0239dff893ef9f15",
+        sha256 = "534fa658bd845fd974b50b10f444d392dfd0d93768c4a51b61263fd37d851c40",
+        strip_prefix = "boringssl-b9232f9e27e5668bc0414879dcdedb2a59ea75f2",
```

Comment:
why is it upgraded?

Reply:
boringssl is upgraded since the boringssl that grpc uses at bazel/brpc_deps.bzl is upgraded. See: https://github.com/grpc/grpc/blob/2717aa8c9b768de99c811628af4e8c30d6c2da4f/bazel/grpc_deps.bzl#L198-L209